source/content/symlinks-assumed-write-access.md

```diff
@@ -43,10 +43,10 @@ The following is for Mac and Linux only. Windows users may refer to Microsoft do
 1. Move the directory you want to replace with a symlink. This serves to both back up any data that may otherwise be lost, and to prevent the symlink from being nested inside the existing directory:
 
     ```bash{promptUser: user}
-    mv ./wp-content/path/plugin-expects-write-to ~/backups/
+    mv ./wp-content/path/plugin-expects-write-to $_ENV['HOME']/tmp/backups
```

Comment:
How are these two versions more or less writable than each other?

Reply:
FYI This is the reader's local computer, so I'm not sure what @joshualieb was getting at tin the original issue.